### PR TITLE
Bugfix/purgecss extractors

### PR DESCRIPTION
**Pull request by yoktav**

### CODE REVIEW

**Comment by yoktav** [comment]
Closes #247

File: config/purgecss.config.js [hunk 1]
```diff
@@ -4,4 +4,13 @@ export default {
...
   variables: true,
+  whitelistPatterns: [/^is|has-/],
+  extractors: [
+    {
+      extractor: (content) => {
+        return content.match(/[a-z-_\\@%0-9]+/g) || []; // Compare class names including '@', '%' and '\'
+      },
+      extensions: ['css', 'html'],
+    },
+  ],
```
Comment (hsnaydd) [inline config/purgecss.config.js]:
```suggestion
  whitelistPatterns: [/is-[\S]*/g, /has-[\S]*/g],
  // Allow to use special characters in class names like `@, :, %`
  defaultExtractor: (content) => content.match(/[\w-/:%@]+(?<!:)/g) || [],
```

**Review by hsnaydd — CHANGES_REQUESTED** [review]

File: config/purgecss.config.js (hunk 1 above)
Comment (yoktav) [reply config/purgecss.config.js]: Don't we need to add \\\ to `/[\w-/:%@]+(?<!:)/g` -> `/[\w-\\/:%@]+(?<!:)/g`

Comment (hsnaydd) [reply config/purgecss.config.js]: No, It will work in this way.

Comment (yoktav) [reply config/purgecss.config.js]: I checked and it seems working in that way too, thank you 😄

**Review by hsnaydd — APPROVED** [review]

**Comment by hsnaydd** [comment]
Thank you @yoktav 🎉

**Pull request merged**